src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.HostAnalyzerInfo.cs

```diff
@@ -90,67 +90,64 @@ static ImmutableHashSet<object> GetFeaturesAnalyzerReferenceIds(HostDiagnosticAn
                     return builder.ToImmutable();
                 }
             }
+        }
 
-            private sealed class HostAnalyzerStateSets
+        private sealed class HostAnalyzerInfo
```

Comment:
view with whitespace off.  this moved up one level of nesting so other callers could talk directly to this.